### PR TITLE
Support workflow_dispatch Inputs Type and Options

### PR DESCRIPTION
**Pull request by dizys**
## Reference
GitHub Actions docs on [on.workflow_dispatch.inputs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs)

## Workflow Example

```yaml
on:
  workflow_dispatch:
    inputs:
      logLevel:
        description: 'Log level'
        required: true
        default: 'warning' 
        type: choice # supported in this PR
        options: # supported in this PR
        - info
        - warning
        - debug 
      tags:
        description: 'Test scenario tags'
        required: false 
        type: boolean # supported in this PR
      environment:
        description: 'Environment to run tests against'
        type: environment # supported in this PR
        required: true 
```

### CODE REVIEW

File: src/lib/workflow.ts [hunk 1]
```diff
@@ -22,6 +22,8 @@ export interface OnPaths {
   "paths-ignore"?: string[];
 }
 
+export type WorkflowDispatchInputsType = 'boolean' | 'number' | 'string' | 'choice';
```
Comment (cschleiden) [inline src/lib/workflow.ts:25]: Couple things:
- there is no support for `number` for `workflow_dispatch`
- there _is_ support for an `environment` input type

**Review by cschleiden — COMMENTED** [review]
Thanks for sending this 🙇 

I had a couple comments about the supported inputs

File: src/lib/workflow.ts (hunk 1 above)
Comment (dizys) [reply src/lib/workflow.ts]: Fixed. I got confused over `workflow_call` which has a `number` option. 😅

**Review by cschleiden — APPROVED** [review]
Thank you!

**Comment by cschleiden** [comment]
I've published the package with this change.

**Pull request merged**